src/ex4/server/services/item_manager.js

```diff
@@ -1 +1,73 @@
-// The ItemManager should go here. Remember that you have to export it.
\ No newline at end of file
+import { readFile, writeToFile } from '../helper/fileHandler.js'
+import { isPokemon, isListOfPokemons, pokemonExistsInTodoList } from '../helper/pokemonHelper.js'
+import { getPokemon, getAllPokemons } from '../clients/pokemon_client.js'
+
+class ItemManager {
+
+    constructor() {
+        this.filePath = "./server/data/data.json" //Why is this not working??
+    }
+
+    async getItems(req, res) {
+        try {
+            const items = await readFile("./server/data/data.json")
+            res.status(200).json(items)
+        } catch (error) {
+            console.log(error)
+            return []
+        }
+    }
+
+    async postItem(req, res) {
+        try {
+            let items = await readFile("./server/data/data.json")
+            if (!items) items = []
+            let newItem = req.body.item
+            if (isListOfPokemons(newItem)) {
+                let listOfPokemons = newItem.split(',');
+                listOfPokemons = await getAllPokemons(listOfPokemons)
+                listOfPokemons.forEach(item => {
+                    items = [...items, `Catch ${item}`]
```

Comment:
This is not optimal complexity wise..
you have array functions for this, you can do `items.push('Catch ${item}');`

but if you really want to simplify it you can just do:
`items.push(listOfPokemons)`